### DISCUSSION TITLE
Web Version

### DISCUSSION THREAD

**Discussion by ghost** [robertodoering/harpy#733] [Ideas]
Since the web version of Twitter have same exact problems as the mobile one, could we port the project with some changes to desktop version as well?
it's flutter, so maybe possible?

**Comment by robertodoering** [comment]
It's definitely possible to build it for the web & desktop but would require a significant amount of ui changes for it to make sense. If anyone wants to give it a shot feel free to do so.

**Comment by dnestico** [comment]
+1. If someone built a web version of harpy I would 100% use it

**Comment by killabit** [comment]
Someone def should do this..
